packages/next/server/config-shared.ts

```diff
@@ -507,6 +508,7 @@ export const defaultConfig: NextConfig = {
   outputFileTracing: true,
   staticPageGenerationTimeout: 60,
   swcMinify: false,
+  output: !!process.env.NEXT_PRIVATE_STANDALONE ? 'standalone' : undefined,
```

Comment:
Is this env var still used? I can't find where it gets assigned

Reply:
I think it was mainly used for testing and to match the other configs